.changeset/dirty-rivers-tickle.md

```diff
@@ -0,0 +1,8 @@
+---
+"@sveltejs/adapter-cloudflare-workers": minor
+"@sveltejs/adapter-cloudflare": minor
+"@sveltejs/adapter-netlify": minor
+"@sveltejs/kit": minor
```

Comment:
```suggestion
"@sveltejs/adapter-cloudflare-workers": patch
"@sveltejs/adapter-cloudflare": patch
"@sveltejs/adapter-netlify": patch
"@sveltejs/kit": patch
```

Reply:
Thank you!
Sorry, it's my first contribution, so I wasn't sure and the change seamed _minor_ ;)

Reply:
No worries! Here's where the concept is taken from: https://semver.org/